names/names.py

```diff
@@ -11,10 +11,14 @@
 f.close()
 
 duplicates = []
-for name_1 in names_1:
-    for name_2 in names_2:
-        if name_1 == name_2:
-            duplicates.append(name_1)
+# for name_1 in names_1:
+#     for name_2 in names_2:
+#         if name_1 == name_2:
+#             duplicates.append(name_1)
+a_set = set(names_1)
+b_set = set(names_2)
```

Comment:
nice you can also use intersection or a for loop with an if all of these would be O(n)